### ISSUE TITLE
Investigate TensorFlow warning in NPO about IndexedSlices

### ISSUE THREAD

**Issue by ryanjulian** [rlworkgroup/garage#289]
```sh
/home/rjulian/miniconda2/envs/garage-joblib/lib/python3.6/site-packages/tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
```

I think this has something to do with the design of our loss function in NPO. 

* Determine if we can address the warning
* If we can't address the warning, determine if it's actually a problem
* If it's not a (big) problem, I think we can find a way to suppress it
